lib/Dialect/FIRRTL/FIRRTLOps.cpp

```diff
@@ -3360,6 +3360,40 @@ void RegResetOp::getAsmResultNames(OpAsmSetValueNameFn setNameFn) {
   return forceableAsmResultNames(*this, getName(), setNameFn);
 }
 
+//===----------------------------------------------------------------------===//
+// FormalOp
+//===----------------------------------------------------------------------===//
+
+LogicalResult
+FormalOp::verifySymbolUses(::mlir::SymbolTableCollection &symbolTable) {
+  auto referencedModule = symbolTable.lookupNearestSymbolFrom<FModuleLike>(
```

Comment:
Do we want to allow an external module? I think it's reasonable to restrict this to FModuleOp.